radix-engine/src/wasm/wasm_instrumenter.rs

```diff
@@ -1,16 +1,18 @@
-use moka::sync::Cache;
 use radix_engine_interface::crypto::hash;
-use std::sync::Arc;
+use sbor::rust::cell::RefCell;
+use sbor::rust::collections::HashMap;
+use sbor::rust::sync::Arc;
 
 use crate::types::*;
 use crate::wasm::{WasmMeteringConfig, WasmModule};
 
 pub struct WasmInstrumenter {
-    cache: Cache<(Hash, Hash), Arc<Vec<u8>>>,
+    cache: RefCell<HashMap<(Hash, Hash), Arc<Vec<u8>>>>,
```

Comment:
I believe this will panic if multiple threads try to lock it at the same time - as can happen in the Node when we are accepting multiple mempool submissions at once.

Can we wrap it in a Mutex instead of a RefCell?